projects/plugins/jetpack/modules/carousel/jetpack-carousel.css

```diff
@@ -340,13 +340,14 @@ div.jp-carousel-buttons a.jp-carousel-commentlink {
 /** Title and Desc Start **/
 .jp-carousel-info-footer .jp-carousel-photo-title-container {
 	flex-basis: 50vw;
```

Comment:
By no means a blocking issue, but I realized it's quite confusing that this `.jp-carousel-photo-title-container` class is used on what's actually the _caption_ underneath the photo. We could probably clean that up in a follow up.